src/domain/ldap/services/class.ldap.php

```diff
@@ -116,7 +116,7 @@ public function bind($username='', $password=''){
             }
             $passwordBind=$password;
 
-            $bind = ldap_bind($this->ldapConnection, $usernameDN, $passwordBind);
+            $bind = ldap_bind($this->ldapConnection, $this->bindUser, $this->bindPassword);
```

Comment:
Greetings
I have to change this because the "$username" is the user that have to login.
In this method ldap_bind you have to inform a user admin or with read permission to bind in your AD
In my configuration.php I informed the full DN of my user admin to perform the bind on var $bindUser

Example that work for me
![image](https://user-images.githubusercontent.com/85897368/204807426-afe9fe2c-3b48-405c-b476-56b4d29647f5.png)

Reply:
I understand what you are trying to do, but this change breaks ldap for active directory users and everyone who doesn't use a central admin user to bind.